### PR TITLE
Update meta tags to use twitter card "summary_large_image"

### PR DESCRIPTION
**Pull request by Mikek2252**
This addresses https://github.com/mainmatter/mainmatter.com/issues/1961 and adds the required meta tags to show the twitter posts as desired.

### CODE REVIEW

**Comment by marcoow** [comment]
deploy preview is here: https://deploy-preview-1964--objective-northcutt-37494c.netlify.app

**Review by marcoow — APPROVED** [review]
🎉

File: src/components/global/meta.njk [hunk 1]
```diff
@@ -38,6 +38,8 @@
 {%- endif -%}
 
 {%- if og.image -%}
+  <meta name="twitter:card" content="summary_large_image">
+  <meta name="twitter:title" content="{{ pageTitle }}">
```
Comment (marcoow) [inline src/components/global/meta.njk:42]:
```suggestion
  <meta name="twitter:title" content="{{ pageTitle }}">
  <meta name="twitter:site" content="@mainmatter">
```

**Review by marcoow — COMMENTED** [review]
I'd add 1 more tag?

**Review by oscard0m — APPROVED** [review]

**Pull request merged**